### DISCUSSION TITLE
Budibase CLI setup.... if you have an arm processor....

### DISCUSSION THREAD

**Discussion by RASS00** [Budibase/budibase#5356] [General]
![image](https://user-images.githubusercontent.com/82243352/162827379-c7e9ce1c-1630-4978-8ccd-42ee7f316a3e.png)
Is this supposed to say AMD64 or ARM64?

**Comment by melohagan** [comment]
Hi @RASS00 

It is supposed to say **AMD64**

Effectively, if you have an ARM based machine, you need to emulate the AMD64 architecture for the couchdb-service to run.

**Reply by RSPont** [reply]
Huh, neat, thanks!
